fe/src/main/java/org/apache/doris/planner/HashJoinNode.java

```diff
@@ -53,8 +54,8 @@
 
     private final TableRef     innerRef;
     private final JoinOperator joinOp;
-    // conjuncts of the form "<lhs> = <rhs>", recorded as Pair(<lhs>, <rhs>)
-    private  List<Pair<Expr, Expr>> eqJoinConjuncts;
+    // predicates of the form "<lhs> = <rhs>", recorded as Pair(<lhs>, <rhs>)
```

Comment:
why you change conjuncts to predicates?
Better to keep it conjuncts, becuase it means `and` and same with `otherJoinConjuncts`